gammapy/irf/background.py

```diff
@@ -135,6 +135,64 @@ def to_2d(self):
     def peek(self, figsize=(10, 8)):
         return self.to_2d().peek(figsize)
 
+    def plot_at_energy(self, energy=None, ax=None, add_cbar=True, ncols=3, **kwargs):
+        """ Plot the background rate in Field of view co-ordinates at a given energy.
+
+        Parameters
+        -----------
+        energy : `~astropy.units.Quantity`
+            list of Energy
+        ax: `~matplotlib.axes.Axes`, optional
+            Axis
+        add_cbar : bool
+            Add color bar?
+        ncols : int
+            Number of columns to plot
+        **kwargs : dict
+            Keyword arguments passed to `~matplotlib.pyplot.pcolormesh`.
+        """
+        import matplotlib.pyplot as plt
+
+        n = len(energy)
+        cols = min(ncols, n)
+        rows = 1 + (n - 1) // cols
+        width = 12
+        figsize = (width, width * rows / cols)
+
+        fig, axes = plt.subplots(
+            ncols=cols,
+            nrows=rows,
+            figsize=figsize,
+            gridspec_kw={"hspace": 0.2, "wspace": 0.3},
+        )
+
+        x1 = self.axes[1]
+        x2 = self.axes[2]
+
+        x = x1.edges.value
+        y = x2.edges.value
+        X, Y = np.meshgrid(x, y)
+
+        for i, ee in enumerate(energy):
+            if len(energy) == 1:
+                ax = axes
+            else:
+                ax = axes.flat[i]
+            bkg = self.evaluate(energy=ee)
+            Z = bkg.squeeze().value
+            im = ax.pcolormesh(X, Y, Z, **kwargs)
+            ax.set_xlabel(x1.name + " [" + str(x1.unit) + "]")
```

Comment:
You can use `MapAxis.format_plot_xaxis(ax=ax)` here, this will take care of everything...